include/dukglue/detail_primitive_types.h

```diff
@@ -45,6 +45,13 @@ namespace dukglue {
 		DUKGLUE_SIMPLE_VALUE_TYPE(int32_t, duk_is_number, duk_get_int, duk_push_int, value)
 		DUKGLUE_SIMPLE_VALUE_TYPE(int64_t, duk_is_number, duk_get_number, duk_push_number, value) // have to cast to double
 
+#ifdef __APPLE__
```

Comment:
Why just for `__APPLE__` This seems handy for most platforms.

Reply:
I've tested the library on Windows (MSVC), Linux (GCC/Clang) and macOS (Clang) and this is a macOS-only bug: all the other platforms support `size_t` and `time_t` functions. That's why I put the `#ifdef __APPLE__` there, to add this missing feature to Apple platforms and prevent all other (working) platforms from generating redefinition errors.

Reply:
If you find any other platforms for which `time_t` and `size_t` functions don't work, just let me know and I'll update this PR :)